install-spotify.sh

```diff
@@ -251,4 +241,46 @@ uninstall () {
     fi
 }
 
+check_arch () {
+	arch=$(arch)
```

Comment:
Again several lines of broken indentation.  Just view the pull request on github and you will see :-)